spec/requests/api/moves_controller_update_v2_spec.rb

```diff
@@ -184,6 +185,35 @@
       end
     end
 
+    context 'when trying to update from_location and to_location' do
+      let(:new_from_location) { create :location }
+      let(:new_to_location) { create :location }
+
+      let(:move_params) do
+        {
+          type: 'moves',
+          attributes: {
+            status: 'requested',
+          },
+          relationships: {
+            from_location: { data: { type: 'locations', id: new_from_location.id } },
+            to_location: { data: { type: 'locations', id: new_to_location.id } },
+          },
+        }
+      end
+
+      it 'does not affect both from_location and to_location' do
+        do_patch
+
+        move.reload
```

Comment:
I think you can remove these two lines? .. or is there a reason why the endpoint is called twice?

Reply:
ops, I thought I've removed it,
good spot thanks!